recipes/azure-storage-cpp/all/conanfile.py

```diff
@@ -40,84 +38,119 @@ def _minimum_cpp_standard(self):
     @property
     def _minimum_compiler_version(self):
         return {
-            "gcc": "5",
+            "gcc": "6",
             "Visual Studio": "14",
+            "msvc": "190",
             "clang": "3.4",
             "apple-clang": "5.1",
         }
 
-    def requirements(self):
-        self.requires("cpprestsdk/2.10.18")
-        if self.settings.os != "Windows":
-            self.requires("boost/1.76.0")
-            self.requires("libxml2/2.9.10")
-            self.requires("util-linux-libuuid/2.39")
-        if self.settings.os == "Macos":
-            self.requires("libgettext/0.20.1")
-
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
-
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-
-        self._cmake.definitions["CMAKE_FIND_FRAMEWORK"] = "LAST"
-        self._cmake.definitions["BUILD_TESTS"] = False
-        self._cmake.definitions["BUILD_SAMPLES"] = False
-        if not self.settings.compiler.cppstd:
-            self._cmake.definitions["CMAKE_CXX_STANDARD"] = self._minimum_cpp_standard
-
-        if self.settings.os == "Macos":
-            self._cmake.definitions["GETTEXT_LIB_DIR"] = self.deps_cpp_info["libgettext"].lib_paths[0]
-
-        self._cmake.configure(build_folder=self._build_subfolder)
-        return self._cmake
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=self.export_sources_folder)
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("cpprestsdk/2.10.19", transitive_headers=True, transitive_libs=True)
+        # LibXML2 v2.12+ from CCI is not compatible. Needs to be built with -DLIBXML_OUTPUT_ENABLED:
+        # https://github.com/GNOME/libxml2/commit/5d2dbe79fae5618d0bd2a7e52be9cf98b11d98ad
+        self.requires("libxml2/2.11.5", transitive_headers=True, transitive_libs=True)
```

Comment:
I'm guessing this is still the case?

Reply:
Does not look like it. Removed the comment.